### PR TITLE
Beminfo 790

### PR DESCRIPTION
**Pull request by tormozz48**
Support cocaine storage as library examples holder
@tavriaforever @gela-d

### CODE REVIEW

File: configs/common/app.json [hunk 1]
```diff
@@ -16,14 +16,13 @@
         "stdout": "logs/output.log",
         "stderr": "logs/errors.log"
     },
+    "storage": {
+        "cocaine": {
```
Comment (tadatuta) [inline configs/common/app.json]: should it be in this repo?

File: src/blocks/server.blocks/storage/storage.node.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    EventEmitter = require('events').EventEmitter,
+    Client = require('cocaine').Client;
+
+modules.define('storage', ['config', 'logger'], function(provide, config, logger) {
+
+    logger = logger(module);
+
+    var OPTIONS = {
+            NAME_SPACE: 'defaultnamespace',
+            HOST: 'apefront.tst.ape.yandex.net',
```
Comment (tadatuta) [inline src/blocks/server.blocks/storage/storage.node.js]: :(

**Comment by tormozz48** [comment]
@tadatuta I came to conclusion that you are right about this code. It should be on more high level.
So I made corresponded changes in code

Also you can see pr for similar branch for lego-site.

**Comment by tadatuta** [comment]
lgtm

**Comment by tormozz48** [comment]
Please do not merge this pr! Storage should be replaced for any other. Cocaine storage is 
inappropriate for our tasks

**Comment by tormozz48** [comment]
This is invalid pr. I will close it

**Pull request closed without merging**